src/Components/NavBar/Menus/NotificationsMenu.tsx

```diff
@@ -72,43 +77,53 @@ const NotificationsMenu: React.FC<NotificationsMenuQueryResponse> = props => {
   )
 }
 
-export const NotificationsMenuQueryRenderer: React.FC = () => {
+export const NotificationsQueryRenderer: React.FC<{
+  render: LoadProgressRenderer<any>
+}> = ({ render }) => {
   const { relayEnvironment } = useContext(SystemContext)
 
   return (
-    <Menu title="Actvity">
-      <QueryRenderer<NotificationsMenuQuery>
-        environment={relayEnvironment}
-        query={graphql`
-          query NotificationsMenuQuery {
-            me {
-              followsAndSaves {
-                notifications: bundledArtworksByArtist(
-                  sort: PUBLISHED_AT_DESC
-                  for_sale: true
-                  first: 10
-                ) @connection(key: "WorksForYou_notifications") {
-                  edges {
-                    node {
-                      href
-                      summary
-                      artists
-                      published_at(format: "MMM DD")
-                      image {
-                        resized(height: 40, width: 40) {
-                          url
-                        }
+    <QueryRenderer<NotificationsMenuQuery>
+      environment={relayEnvironment}
+      query={graphql`
+        query NotificationsMenuQuery {
+          me {
+            followsAndSaves {
+              notifications: bundledArtworksByArtist(
+                sort: PUBLISHED_AT_DESC
+                for_sale: true
```

Comment:
@mzikherman - does this flag mean _only_ for sale items, or is it inclusive?

Reply:
@mzikherman - ping

Reply:
Yea, this param means only for sale items.